tests/test_bs.py

```diff
@@ -417,7 +437,7 @@ def test_add_client(add_client):
     assert len(hits) == 1, "Could not find new Client bootstrap message"
 
 
-def test_gossip(setup_clients):
+def test_gossip(setup_clients, add_curl):
```

Comment:
isn't it better to add it as a precondition to a more basic fixture like the the one that sets the namespace?

Reply:
I added it here because there are few tests that don't need the curl.